uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -506,15 +530,16 @@ define([
      */
     function setCalendarProps(contactID, calendar) {
       var leaveRequest,
-        indexedByMonth = vm.months.map(function () {
-          return [];
+        monthObject,
+        indexedByMonth = _.map(vm.months, function (month) {
+          return _.extend(month, {
+            data: []
```

Comment:
also, anyway this name `indexedByMonth` doesn't seem to be representative of what the variable is holding, i don't see anything being indexed by anything